src/clj/commiteth/eth/web3j.clj

```diff
@@ -25,4 +29,26 @@
                       {:password password :file-path file-path})))))
 
 (defn create-web3j []
-  (Web3j/build (HttpService. (eth/eth-rpc-url))))
+  (Web3j/build (HttpService. (env :eth-rpc-url "http://localhost:8545"))))
+
+(defn get-signed-tx [gas-price gas-limit to data]
```

Comment:
How about placing this in `commiteth.eth.core`? To me it is more a feature about Ethereum, even if we use the tool `web3j` to solve it.

Reply:
I'd then say we banish `web3j` ns altogether. There's not much more in there anyway. What do you think?

Reply:
Agree! :)

Reply:
Removed.